spec/fixtures/local.yml

```diff
@@ -3,3 +3,5 @@ ldap:
   hostname: "ldap.example.com"
   port: 389
   base: "ou=users,dc=example,dc=com"
+unknown:
+  enabled: true
```

Comment:
It's an invalid configurable option since the picked config file for the defaults doesn't have this. I missed explicitly checking that it's not set :)